solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -19,6 +19,8 @@ import "./libraries/Wallets.sol";
 import "./DKGValidator.sol";
 import "@keep-network/sortition-pools/contracts/SortitionPool.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
+import {IRandomBeacon} from "@keep-network/random-beacon/contracts/RandomBeacon.sol";
+import {IRandomBeaconConsumer} from "@keep-network/random-beacon/contracts/libraries/Callback.sol";
```

Comment:
Non-blocking this PR but we could extract `IRandomBeaconConsumer` interface to its own `*.sol` file. Importing an interface from a library that is supposed to be internal to the random beacon code is a code smell.

Reply:
I don't like that the interface sits in the library's file. But extracting it seems inconsistent with other interfaces that are defined in `RandomBeacon.sol`. How about we extract all the interfaces to a `interfaces/` directory, separate `*.sol` file each?

Reply:
There is a difference between what's in the internal library and what's in the contract `WalletRegistry` is supposed to have an address to. Two options are equally good for me from the external perspective:
- Add `api/` directory, have there `IRandomBeacon.sol` and `IRandomBeaconConsumer.sol`
- Inside `RandomBeacon.sol` have `IRandomBeacon` and `IRandomBeaconConsumer` defined.

However, the second option sucks from `Callback.sol`'s perspective because it imports something from a file it was supposed to be imported from.

So I'd go with `api` (not `interfaces`, let's make it concise) and separate files as you suggested.

Reply:
15022421ca49ac41811cd6b9d26cb9b160aa0d80